orttraining/orttraining/python/training/api/lr_scheduler.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# lr_scheduler.py
+
+from onnxruntime.capi import _pybind_state as C
+
+
+class LRScheduler:
+    """
+    Class for running Learning Scheduler Step in Training.
+    This class is a wrapper of LRScheduler Class.
```

Comment:
Let's provide more details here. What kind of LR scheduler is this (linear, ...)? How should the user use it?

Also, let's avoid reference to the C++ class by saying that this is a wrapper of the underlying C++ class. Python users may not know the C++ class.

For example:

```py
"""Linearly updates the learning rate in the optimizer

    The linear learning rate scheduler decays the learning rate by linearly updated
    multiplicative factor from the initial learning rate set on the training session to 0. The decay
  is performed after the initial warm up phase where the learning rate is linearly incremented
  from to the initial learning rate provided.

    Args:
        optimizer (:obj:`training_api.Optimizer`): User's onnxruntime training Optimizer
        warmup_step_count (int): The number of steps in the warm up phase.
        total_step_count (int): The total number of training steps.

```